frontend/src/components/mine/Variances/MineVarianceTable.js

```diff
@@ -39,6 +39,20 @@ const propTypes = {
   }),
 };
 
+const codeSorter = (code_one, code_two) => {
```

Comment:
Can this be moved to 'utils' since it's the same in both files, also update the name to be a little more clear ie: `complinceCodeSorter`